buildSrc/build.gradle.kts

```diff
@@ -0,0 +1,16 @@
+repositories {
```

Comment:
have you looked into composite builds instead of buildsrc?

Reply:
Yes, and I think it'll be easier for now to go with buildSrc.